style.css

```diff
@@ -116,7 +127,119 @@ h2 {
 }
 
  .fruit-image {
-    margin-left: 61px;
+    margin-left: 47px;
+}
+
+.socialmedia-icons {
+    position: absolute;
+    bottom: 0;
+    left: 87px;
+    color: #454341;
+    font-size: 20px;
+    max-width: 218px;
+}
+
+.fab {
+    margin-right: 20px;
+    
+}
+
+.slider-container {
+    display: flex;
+    justify-content: center;
+    align-items: baseline;
+    margin-top: 64px;
+}
+
+.arrows {
+    margin: 2%;   
+}
+
+.left-arrow-circle {
+    padding: 7px;
+    color: #FFFFFF;
+    background: #FA6400;
+    border-radius: 50px;
+    float: left;
+}
+
+.right-arrow-circle {
+    padding: 7px;
+    background: #FA6400;
+    color: #FFFFFF;
+    border-radius: 50px;
+    float: right;
+}
+   
+
+.image-slider {
+    display: flex;
+    align-items: center;
+    max-width: 521px;
+}
+
+.left-fruit {
+    background-color: #FA6400;
+    border-radius: 7px 64px 7px 64px;
+    width: 125.43px;
+    height: 124.12px;
```

Comment:
Please, remove the decimal values, use integer only, for example: `height: 124px;`